sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldNotBeIgnored.MsTest.cs

```diff
@@ -1,9 +1,9 @@
-﻿using System;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace Tests.Diagnostics
+namespace Tests.Diagnostics.TestMethods
 {
-    class MsTestClass
+    [TestClass]
```

Comment:
I modified the existing test code so that the test code is valid, so it can be cut-and-paste into a test project and executed to check that it is working as expected.